src/components/WelcomeStepsModal/WelcomeModalPostalCode.tsx

```diff
@@ -33,10 +35,12 @@ export const WelcomeModalPostalCode: React.FC = () => {
   const classes = { ...useCommonWelcomeModalStyles(), ...useStyles() };
   const dispatch = useThunkDispatch();
   const history = useHistory();
-  const [postCode, setPostCode] = useState("");
+  const postCode = useSelector((state: State) => state.app.intro.postCode);
+  const checked = useSelector((state: State) => state.app.intro.isPrivacyChecked);
+  // const [postCode, setPostCode] = useState("");
```

Comment:
The comments should be removed (also three lines below), but otherwise it looks good to me.